### DISCUSSION TITLE
Meshroom graphics card

### DISCUSSION THREAD

**Discussion by potblender** [alicevision/Meshroom#2129] [Q&A]
Looking for a new laptop capable of running Meshroom. I undeestand Nvidia GeForce CUDA is the best to run Meshroom. I'm looking at the three system below but I'm unsure if the Ryzen/AMD/Integrated cards are CUDA?

Ryzen 7 5800H Integrated DDR4 SDRAM graphics card

i5-11400HIntel Processor 8GBDDR4 Memory GeForce RTX™ 2050NVIDIA Laptop Graphics 512GB storage

Ryzen 5 5600HAMD Processor AMD RadeonGraphics

Any advice or recommendations most welcome!

**Comment by natowi** [comment]
Only Nvidia GPUs use CUDA like the GeForce RTX 2050
But I would recommend you to base your decision not only on the requirement of one software component. Also think on how else you want to use your setup and what the trade-of is

**Reply by potblender** [reply]
Thanks for the reply.
So an Intel Iris Xe Graphics graphics card on a  intel i7 would be no good either?
Had a go at photogrammetry using Meshroom on my now kapputt laptop (Asus N73?) in which I had to put extra RAM in, and managed  quite well. Looking for a updated model/set up to try and to produce a 3D print of my project.
I've seen what the minimum/recommended specs are required to run Meshroom (the depthmap being an issue?) but those specs were from a couple years back and I'm wondering of the compatible cards/processors etc were.
Looking for something like a 16gb RAM 1TB Storage Nvidia graphics I think...if you've any idea!? Thanks again!

**Reply by natowi** [reply]
When buying a new computer I would not have less than 16GB Ram, 1TB is fine too. If you decide on Nvidia (which is required for Meshroom), you don´t need the latest model, an older 1060 or 1070 works great. The compute capability has to be equal or greater than 3.5 - here are lists to check your preferred GPU: https://developer.nvidia.com/cuda-gpus
